[2023-12-21 02:21:18,905] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla_postgres scheduled__2023-12-11T00:00:00+00:00 [queued]>
[2023-12-21 02:21:19,087] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla_postgres scheduled__2023-12-11T00:00:00+00:00 [queued]>
[2023-12-21 02:21:19,109] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-21 02:21:19,117] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-21 02:21:19,128] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-21 02:21:19,306] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): crear_tabla_postgres> on 2023-12-11 00:00:00+00:00
[2023-12-21 02:21:19,414] {standard_task_runner.py:52} INFO - Started process 192 to run task
[2023-12-21 02:21:19,561] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_con_conexion_postgres', 'crear_tabla_postgres', 'scheduled__2023-12-11T00:00:00+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/dag_postgres_database.py', '--cfg-path', '/tmp/tmpu_mg8jd8', '--error-file', '/tmp/tmphsxb6548']
[2023-12-21 02:21:19,639] {standard_task_runner.py:80} INFO - Job 121: Subtask crear_tabla_postgres
[2023-12-21 02:21:21,867] {task_command.py:371} INFO - Running <TaskInstance: dag_con_conexion_postgres.crear_tabla_postgres scheduled__2023-12-11T00:00:00+00:00 [running]> on host de0a7ae97836
[2023-12-21 02:22:06,916] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=dag_con_conexion_postgres
AIRFLOW_CTX_TASK_ID=crear_tabla_postgres
AIRFLOW_CTX_EXECUTION_DATE=2023-12-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-11T00:00:00+00:00
[2023-12-21 02:22:07,141] {base.py:68} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-12-21 02:22:07,430] {dbapi.py:231} INFO - Running statement: 
            create table if not exists fin_mundo(
                dt date,
                pais varchar(30)
            )
        , parameters: None
[2023-12-21 02:22:07,554] {postgres.py:94} INFO - NOTICE:  la relación «fin_mundo» ya existe, ignorando

[2023-12-21 02:22:12,075] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_con_conexion_postgres, task_id=crear_tabla_postgres, execution_date=20231211T000000, start_date=20231221T022118, end_date=20231221T022212
[2023-12-21 02:22:12,375] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-21 02:22:12,654] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
